clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -2421,12 +2425,14 @@ protected DelegateConsumerRecordResult delegateConsumerRecord(
 
       // If we are here the message must be produced to local kafka or silently consumed.
       LeaderProducedRecordContext leaderProducedRecordContext;
-
+      // No need to resolve cluster id and kafka url because sep topics are real time topic and it's not VT
       validateRecordBeforeProducingToLocalKafka(consumerRecord, partitionConsumptionState, kafkaUrl, kafkaClusterId);
```

Comment:
One thing not clear to me yet is if we resolve the cluster id and url here, which are supposed to be no-op, a couple integ tests would fail with or without my change. So I'm sure by design, we are not supposed to use kafka url resolver here but I don't know why. 😕

Reply:
Although we can resolve later, could you point out which test will be failing for example?

Reply:
Those are:
IntegTest_1220
```
1. TestPushJobWithEmergencySourceRegionSelection#testNativeReplicationForBatchPushWithEmergencySourceOverride[0](50, 2)
2. TestPushJobWithNativeReplication#testTargetedRegionPushJobFullConsumptionForBatchStore
3. TestPushJobWithSourceGridFabricSelection#testPushJobWithSourceGridFabricSelection[0](50, 2)

```
and
```
IntegTest_1050
1. TestActiveActiveReplicationForIncPush#testAAReplicationForIncrementalPushToRT[0]
```